README.md

```diff
@@ -16,53 +16,62 @@ The Azure Functions Core Tools provide a local development experience for creati
 
 **v2** (master branch): Self-contained cross-platform package
 
-**v3**: Self-contained cross-platform package
+**v3**: (v3.x branch): Self-contained cross-platform package **(recommended)**
 
 ## Installing
 
 ### Windows
 
-To install runtime with npm:
+To download and install with MSI:
 
-**v2**
-```bash
-npm i -g azure-functions-core-tools@2 --unsafe-perm true
-```
+**v3**
+
+- [Windows 64-bit](https://go.microsoft.com/fwlink/?linkid=2135274) - Required for VS Code debugging
+- [Windows 32-bit](https://go.microsoft.com/fwlink/?linkid=2135275)
+
+To install with npm:
 
 **v3**
 ```bash
 npm i -g azure-functions-core-tools@3 --unsafe-perm true
 ```
 
+**v2**
+```bash
+npm i -g azure-functions-core-tools@2 --unsafe-perm true
+```
+
 To install with chocolatey:
 
-"**v2**"
+**v3**
 ```bash
-choco install azure-functions-core-tools
+choco install azure-functions-core-tools-3
 ```
-"**v3**"
+
+**v2**
 ```bash
-choco install azure-functions-core-tools-3
+choco install azure-functions-core-tools-2
 ```
 
-#### Notice: To debug functions under vscode, x64 bitness is required
+*Notice: To debug functions under vscode, x64 bitness is required*
```

Comment:
nit: Should the verbiage be:
`Notice: To debug functions under vscode, the 64-bit version is required`